ui/less/containers.less

```diff
@@ -196,10 +196,11 @@
 
   /* Set the captions in the middle horizontally by default. */
   display: flex;
-  justify-content: center;
+  flex-direction: column;
+  justify-content: flex-end;
 
   /* Set the captions at the bottom by default. */
-  align-items: flex-end;
+  align-items: center;
 
```

Comment:
I've changed the flex-direction for the cues to display on a column so I had to change the flex rules on the container.

That was mainly because I've tested prior to the support of span of p, I'll revert this to see if it still works ok !